### ISSUE TITLE
es5 error

### ISSUE THREAD

**Issue by chenfengjw163** [beatgammit/base64-js#61]
`base64-js@1.5.0` use `const` keyword:
```js
const lookup = []
``` 
compile to es5 has error.
```
The keyword 'const' is reserved
```

**Comment by 502647092** [comment]
can't run this package at java nashorn engine
that not support const

**Comment by aleen42** [comment]
```
xxx from UglifyJs
Unexpected token: keyword (const) [./node_modules/base64-js/index.js:7,0]
```

**Comment by Unsfer** [comment]
Package must be transpiled to es5 before publish

**Comment by chenfengjw163** [comment]
#62

**Comment by feross** [comment]
Released a fix as 1.5.1
